service/resource/deployment/resource_test.go

```diff
@@ -187,110 +189,14 @@ func Test_Resource_Deployment_GetCreateState(t *testing.T) {
 		}
 	}
 	for i, tc := range testCases {
-		result, err := newResource.GetCreateState(context.TODO(), tc.Obj, tc.Cur, tc.Des)
+		deployments, err := newResource.newCreateChange(context.TODO(), tc.Obj, tc.Cur, tc.Des)
 		if err != nil {
-			t.Fatalf("case %d expected '%v' got '%#v'", i+1, nil, err)
-		}
-
-		deployments, ok := result.([]Deployment)
-		if !ok {
-			t.Fatalf("case %d expected '%T', got '%T'", i+1, []Deployment{}, deployments)
-		}
-
-		if len(tc.ExpectedDeployments) != len(deployments) {
-			t.Fatalf("case %d expected %d deployments got %d", i+1, len(tc.ExpectedDeployments), len(deployments))
-		}
-	}
-}
-
-func Test_Resource_Deployment_GetDeleteState(t *testing.T) {
```

Comment:
Why are we removing the delete tests?

Reply:
Because `NewDeletePatch` is noop. Those tests were an unnecessary clutter IMO.

Reply:
Ack.